Known bugs: add concurrency bug (#9)
shamirlabs/docs: docs/testnet/known-bugs.md

# Known bugs

## Stuck at "Fetching your node identity"

> When using the file `docker-compose-with-clients-metrics.yml` or `docker-compose-with-clients.yml`, the Diva client will wait until the Ethereum nodes are synced without being able to start. If that happens, the Diva Operator UI might show a `Fetching your node identity...` message that never resolves. Please, restart the Diva client after your Ethereum clients are synced to solve the issue.

Reviewer: These files don't exist anymore. Since you are changing this section, please update accordingly.

Author: i didnt change this text, and feel underqualified to do it. could you update it? either in this PR or next one

this PR is just adding a header to this section

Author: ping here

Reviewer: I don't have time to do it, but we should only merge a PR if we know things are right. It is as easy as changing it with the files that are in the testnet repo: https://github.com/shamirlabs/diva-alpha-net

If you can't do it, I will do it when I find time.

Author: this code wasn't edited, it's unrelated to the PR. i simply added a header. the real fix is below

## Not receiving any validator key shares

> The current Diva version doesn't require any collateral to receive validator key shares. You will start receiving them simply by remaining online. As new ETH is staked into Diva, validators will be created and you will have a chance to randomly receive some of the new key shares.

Reviewer: ```suggestion
The current Diva version doesn't require any collateral to receive validator key shares. You will start receiving them simply by remaining online. As new ETHs are staked into Diva, validators will be created, and you will have a chance to receive some of the new randomly assigned key shares.
```
